ocaml/sdk-gen/component-test/jsonrpc-client/go/main_test.go

```diff
@@ -50,3 +59,881 @@ func TestLoginSuccess(t *testing.T) {
 		return
 	}
 }
+
+func TestVMGetAllRecords(t *testing.T) {
+	session, err := getSession("test_id2")
+	if err != nil {
+		t.Log(err)
+		t.Fail()
+		return
+	}
+
+	var result map[xenapi.VMRef]xenapi.VMRecord
+	var vmRecord0 = xenapi.VMRecord{
```

Comment:
Could we load those key-value from session.json and convert to VMRecord struct? that we can avoid duplicate content both in go code and json file.